org.eclipse.jdt.core/META-INF/MANIFEST.MF

```diff
@@ -6,6 +6,16 @@ Bundle-Version: 3.40.100.qualifier
 Bundle-Activator: org.eclipse.jdt.core.JavaCore
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
+Import-Package: org.eclipse.core.internal.runtime,
```

Comment:
why importing an internal package?

Reply:
Also why are these imported without a version contraint?

Reply:
> why importing an internal package?

The use of internal packages was not introduced by this PR; it originates from existing code in jdt.core, which already references these internal packages.